dkg/node.go

```diff
@@ -51,13 +51,33 @@ func (n *Node) newRunner(id RequestID, initMsg *Init) (*Runner, error) {
 		config:                n.config,
 	}
 
-	if err := runner.protocol.Start(initMsg); err != nil {
+	if err := runner.protocol.Start(InitOrReshare{Init: initMsg}); err != nil {
 		return nil, errors.Wrap(err, "could not start dkg protocol")
 	}
 
 	return runner, nil
 }
 
+func (n *Node) newResharingRunner(id RequestID, reshareMsg *Reshare) (*Runner, error) {
```

Comment:
maybe use runner as interface?

Reply:
Done in https://github.com/bloxapp/ssv-spec/pull/82/commits/a5908c6a0f04d8ad440b08579ee8b82e3490ac0c